bridges/pangolin-ropsten/bridge/src/service/check.rs

```diff
@@ -103,6 +103,9 @@ async fn run(tracker: &Tracker) -> color_eyre::Result<()> {
         let verified = match client.ethereum().is_verified(&tx_hash, tx_index).await {
             Ok(v) => v,
             Err(e) => {
+                if e.is_restart_need() {
+                    return Err(e.into());
+                }
```

Comment:
Request review from @fewensa

Reply:
This pull request changed subxt version for pangolin-ropsten bridge, now we must use websocket connection. so there make change follow #434

Reply:
https://github.com/darwinia-network/bridger/pull/434/files#diff-396a63349b5c399e41a09700e4e845933805e750b585e93ac6ea37bc80c80663R106